packages/jest-matchers/src/__tests__/matchers-test.js

```diff
@@ -605,12 +605,12 @@ describe('.toHaveProperty()', () => {
     [{a: 0}, 'a', 0],
     [{a: {b: undefined}}, 'a.b', undefined],
     [{a: {b: {c: 5}}}, 'a.b', {c: 5}],
-  ].forEach(([obj, path, value]) => {
+  ].forEach(([obj, keyPath, value]) => {
```

Comment:
@dmitriiabramov I also change the arg in tests from `path` to `keyPath`, if you think this shouldn't be changed. I'll re change it.